.github/workflows/build_test.yml

```diff
@@ -43,7 +44,23 @@ jobs:
             echo "Invalid testing configuration provided"
             exit 1
           fi
-          echo "config=$(echo -n "$config" | base64 -w 0)" >> "$GITHUB_OUTPUT"
+
+          arch=$(echo -n "$config" | yq -r '.architecture // "amd64"')
```

Comment:
This comment relates to the line #237 (`SKIP_ARM: true`): shouldn't we build the arch images depending on the given arch (output of this job) in the hosted case?